src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class ColorSupplier {
+    private int colorNumber;
```

Comment:
```suggestion
    private Random random = new Random();
    private static final int COLOR_NUMBER = 4;
```
Краще логіку рандому стосовно кольору перенести з FigureSupplier до ColorSupplier